edgelet/build/windows/build.ps1

```diff
@@ -1,23 +1,80 @@
 # Copyright (c) Microsoft. All rights reserved.
```

Comment:
File seems to have a mix of LF and CRLF. Please normalize to LF.

Reply:
Fixed. Just curious, why LF in build.ps1 and CRLF in package.ps1?

Reply:
It was originally CRLF but another commit accidentally changed it to LF. I don't particularly care about which one it is, just that it's consistent and doesn't change unnecessarily. So if the rest of the file is CRLF then keep it CRLF. If it's LF then keep it LF.